libs/fairness/src/lib/Controls/BinDialog.styles.ts

```diff
@@ -95,3 +95,17 @@ export const BinDialogStyles: () => IProcessedStyleSet<IBinDialogStyles> =
       }
     });
   };
+
+export const SpinButtonStyles = {
```

Comment:
nit: Is this the same as the one added in MultiCEPlpt.styles.ts? wondering if it makes sense to take it out and put in a common place

Reply:
Good point! Let me create a common SpinButtonStyles so it can be shared!